src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/presubmit/UpdateInterpreterDetailsHandler.java

```diff
@@ -50,6 +55,7 @@ public PreSubmitCallbackResponse<AsylumCase> handle(PreSubmitCallbackStage callb
         if (!interpreterDetailsList.isEmpty()) {
             asylumCase.write(INTERPRETER_DETAILS, interpreterDetailsList);
         }
+        asylumCase = iaHearingsApiService.aboutToSubmit(callback);
```

Comment:
We might need a functional test. It could depends on: [RIA-7977](https://tools.hmcts.net/jira/browse/RIA-7977)